source/stylesheets/_refills-styles.scss

```diff
@@ -376,32 +374,64 @@ pre[class*="language-"] {
   line-height: 1em;
 }
 
-.copy-source {
-  background-color: lighten(#333, 20%);
-  border-radius: 10px;
-  color: #fff;
-  font-size: 0.5em;
-  font-weight: 800;
-  padding: 0.4em 1em;
-  text-transform: uppercase;
-
-  &:hover,
-  &:active {
-    background-color: lighten(#333, 20%);
-    color: #fff;
-  }
-}
-
 .snippets-table {
   display: none;
 
   td {
     border-bottom: 0;
   }
+
+  .snippet:nth-child(4n) {
+    display: none
+  }
 }
 
+
 .snippet {
-  $refills-medium-screen: em(640) !default;
+  .copy-source-container {
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets